plane/src/drone/docker/commands.rs

```diff
@@ -146,6 +191,40 @@ pub fn get_container_config_from_executor_config(
         .map(|(k, v)| format!("{}={}", k, v))
         .collect();
 
+    let binds: Option<Vec<String>> = match (&mount_base, &exec_config.mount) {
+        (_, None) | (_, Some(Mount::Bool(false))) => None,
+        (Some(base), Some(mount_option)) => {
+            let mount_path = match mount_option {
+                Mount::Bool(true) => {
+                    if let Some(key) = key {
+                        base.join(&key.key.name)
+                    } else {
+                        return Err(anyhow::anyhow!(
+                            "Key is required for Bool(true) mount option"
+                        ));
+                    }
+                }
+                Mount::Path(path) => base.join(path),
+                _ => return Err(anyhow::anyhow!("Invalid mount option")),
```

Comment:
Since this is impossible (if the case is `Mount::Bool(false)` we would have fallen through on the first case of the outer `match`, let's use `unreachable!()` here instead.

If you do that, I would also use `Mount::Bool(false)` as the case here instead of `_` because it will be easier for a reader to understand _why_ it is unreachable.